### PR TITLE
More pond UI changes

### PR DESCRIPTION
**Pull request by breville**
Screenshot 2019-11-05 11 34 43

### CODE REVIEW

File: src/demo/models/pond.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const arrangeFish = fishes => {
     );
     const y = randomInt(
       0,
-      constants.canvasHeight - constants.fishCanvasHeight / 2
+      constants.canvasHeight - constants.fishCanvasHeight / 2 - 160
```
Comment (madelynkasula) [inline src/demo/models/pond.js]: what's the purpose of this change?

Comment (breville) [reply src/demo/models/pond.js]: Good question.  It stops fish showing behind the UI.  I updated with a variable and an explanation.
